cmd/.clangd

```diff
@@ -1,2 +1,2 @@
 CompileFlags:
-  Add: -I/usr/include/glib-2.0 -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes -Wno-missing-field-initializers -Wno-unused-parameter
\ No newline at end of file
+  Add: [-I/home/amurray/go/src/github.com/alexmurray/snapd-private/cmd, -I/usr/include/glib-2.0, -Wall, -Wextra, -Wmissing-prototypes, -Wstrict-prototypes, -Wno-missing-field-initializers, -Wno-unused-parameter]
```

Comment:
```suggestion
  Add: [-I/usr/include/glib-2.0, -Wall, -Wextra, -Wmissing-prototypes, -Wstrict-prototypes, -Wno-missing-field-initializers, -Wno-unused-parameter]
```

Reply:
ugh.. lol so embarrassed I didn't notice that before...